app.js

```diff
@@ -117,14 +140,14 @@ function pingCowin({ key, hook, age, districtId, appointmentsListLimit, date, pi
             }
         }
         if (isSlotAvailable) {
+            console.log(Json.stringify(dataOfSlot));
```

Comment:
This `console.log()` would dump all the data on the console which wouldn't look pretty on the terminal. So this would need to be removed.

Reply:
Sure, we can remove.